aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -180,6 +181,15 @@ void trunc_kernel_cuda(TensorIterator& iter) {
   });
 }
 
+void reciprocal_kernel_cuda(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "reciprocal_cuda", [&]() {
+    using acc_t = acc_type<scalar_t, true>;
```

Comment:
nit: I like providing a comment for boolean arguments (`acctype<scalar_t, /*is_cuda=*/true>`) to make it clearer